app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -60,6 +60,8 @@ class SupplementalApplicationPage extends React.Component {
 
     await updateApplicationAction(synchedApplication)
     this.setState({ persistedApplication: synchedApplication })
+    // Redirect to same page to be able to re-pull data from SalesForce
+    window.location.href = appPaths.toApplicationSupplementals(application.id)
```

Comment:
@fddayan @chrisdolendo a couple of initial concerns I have around this: 
1. What if the fields haven't actually been updated by the time we query the API? Is that an issue we've had in the past?
2. This messes with the synchedApplications state piece, and might actually make it obsolete. Does it also mess with the confirmed preferences piece?

Reply:
this seems to work OK, and was also approved as a valid workaround by jesse